lib/karen.coffee

```diff
@@ -117,7 +117,14 @@ class MockNode extends MockElement
   getAttribute: (name) ->
     @attributes[name]
 
-  getElementsByTagName: -> []
+  getElementsByTagName: (name) ->
+    found = []
+    for child in @children
+      if child.tagName.toLowerCase() == name.toLowerCase()
+        found.push(child)
+      if child.tagName.toLowerCase() != 'iframe'
```

Comment:
Limked MDN says: "When called on an HTML element in an HTML document, getElementsByTagName put its argument in lower case before proceeding." However, @Fonsan is sort of correct, but is commenting on wrong PR, as tagName spec says: "HTML DOM returns the tagName of an HTML element in the canonical uppercase form, regardless of the case in the source HTML document." Will fix in #5.